files/opt/mesh/retrieve_ip

```diff
@@ -66,13 +66,5 @@ echo "recieved mesh block:" $MESHBLOCK"/26" >>$LOG
 
 # complete rest of configurations in a reusable script
 /opt/mesh/autoconf $MESHIP
-
-# start some services
-/etc/init.d/babeld start
-/etc/init.d/meshrouting start
```

Comment:
@Juul @paidforby Might be good to leave these here? Do they cause any badness.

I see how they are somewhat futile if we're just going to reboot soon. But idk, things later in the script could fail before reboot fully works, and seems worth having the meshy stuff running even we never get to the reboot.

Said another way: Seems like this PR can accomplish it's goals without removing these two lines. And Occam's Razor.

Reply:
@gobengo there was discussion in #135 about this. The rationale (open to critique) is that:
1. It would be good to have one source of truth for when and how meshrouting is enabled and/or started.
2. Definitely *need* it enabled since we expect to reboot. This is is done [in autoconf](https://github.com/sudomesh/sudowrt-firmware/blob/master/files/opt/mesh/autoconf#L168).
3. To keep from having two sources of truth, we should either move the start commands into `autoconf`or remove them.

I'm doing the latter, but the former seems to be more to your point.